tests/test_modeling_fsmt.py

```diff
@@ -126,7 +134,7 @@ class FSMTModelTest(ModelTesterMixin, GenerationTesterMixin, unittest.TestCase):
     all_generative_model_classes = (FSMTForConditionalGeneration,) if is_torch_available() else ()
     is_encoder_decoder = True
     test_pruning = False
-    test_head_masking = False
+    test_head_masking = True
```

Comment:
I think this can be removed (as from all other places where it is set to `True`) as this is the default on the superclass.

Imo it makes it more readabe to only have items which are not tested (and which ultimately should be), this way from a quick hover on the test file we can understand what is left to do to have all the common tests coverage. Let me know what you think.

Reply:
Thanks, that's a good point. I'll create a PR removing all redundant `test_head_masking = True` from other source code files later today.

Reply:
Thanks, that's a good point. I'll create a PR removing all redundant test_head_masking = True from other source code files later today.

(Also, I did rebase as `jaxlib` was updated in #9831. Now, it should be, hopefully, fine without any errors :))